src/backend/aspen/app/views/validation.py

```diff
@@ -0,0 +1,62 @@
+from typing import Iterable, Set
+
+import sentry_sdk
+from flask import g, make_response, request
+from marshmallow.exceptions import ValidationError
+from sqlalchemy.orm.query import Query
+
+from aspen.app.app import application, requires_auth
+from aspen.app.serializers import ValidateIDsRequestSchema, ValidateIDsResponseSchema
+from aspen.app.views.api_utils import (
+    authz_sample_filters,
+    get_matching_gisaid_ids,
+    get_missing_and_found_sample_ids,
+)
+from aspen.database.models import Sample
+from aspen.error import http_exceptions as ex
+
+
+@application.route("/api/validate/ids")
```

Comment:
Reasonable people can have differing opinions on this, but I think this endpoint falls under the scope of `samples` since it's about validating the existence of samples generally. I would pitch for something under that previous URI umbrella, like `/api/samples/validate_ids` instead

Reply:
good idea, i'll move it over to samples